lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_method.rb

```diff
@@ -212,5 +213,42 @@ def self.cleanup(method_pid, threads)
       threads.each(&:exit)
     end
     private_class_method :cleanup
+
+    def self.get_bodies_and_line_numbers(obj, aem)
```

Comment:
Generally, `get_` as a method prefix is a smell.  it's typically not necessary, or shows that the method name could be more expressive.  In this case `bodies_and_line_numbers` is sufficient or perhaps `bodies_and_line_numbers_from_object`.